source/manual/howto-modify-change-note.html.md

```diff
@@ -1,13 +1,14 @@
 ---
 owner_slack: "#govuk-2ndline"
-title: Modify a change note in Publishing API
+title: Modify a change note in Publishing API or Whitehall
 section: Publishing
 layout: manual_layout
 parent: "/manual.html"
 ---
 
-Spelling mistakes can creep into change notes, and we are often asked
-to correct them.
+Spelling mistakes can creep into change notes, and we are often asked to correct them. The instructions below cover doing this task in the Publishing API and in Whitehall - if your change note lives in Whitehall and is updated in the Publishing API it could be overwritten later.
```

Comment:
Not sure I understand what's meant by "if your change note lives in Whitehall and is updated in the Publishing API it could be overwritten later." - as in if you change it Publishing API but not Whitehall, the change not in Publishing API will overwritten by the note in whitehall?

Reply:
Yep, that's the impression I got from Richard. My understanding is tenuous, tbh!